client/my-sites/customer-home/components/full-screen-launchpad.tsx

```diff
@@ -59,41 +70,65 @@ export const FullScreenLaunchpad = ( { onClose }: { onClose: () => void } ): JSX
 		dispatch( requestSite( siteId ) );
 	};
 
+	const {
+		data: { checklist },
+	} = useSortedLaunchpadTasks( siteSlug, checklistSlug, launchpadContext );
+
 	if ( isDismissed ) {
 		return null;
 	}
 
-	const isAllTasksCompleted = hasChecklist && numberOfSteps > 0 && completedSteps === numberOfSteps;
+	const launchSiteTask = checklist?.find( ( task: Task ) =>
+		[ 'site_launched', 'blog_launched', 'link_in_bio_launched' ].includes( task.id )
+	);
```

Comment:
Ah, this is too bad, having to have an exhaustive list of "launch tasks" on the front end. Maybe it'd be nice for the backend tasks to have a `is_launch_task` boolean flag 🤷

Reply:
Like this one? hahaha

![image](https://github.com/user-attachments/assets/df493af2-29ed-4092-a36a-a8fa10db7a6e)

Reply:
omg! 🤣 our forebears thought of everything